### ISSUE TITLE
Getting NaN in Mel Loss during the first few epochs for first training

### ISSUE THREAD

**Issue by iAdityaVishnu** [yl4579/StyleTTS2#327]
I am noticing NAN in mel losses. The nan is coming in the first few epochs itself. Does anyone know how can this be solved?

This on the master brand code that I am training using two H100.

@yl4579

**Comment by iAdityaVishnu** [comment]
![Image](https://github.com/user-attachments/assets/a4c23af2-336e-4a46-bfba-267699b66812)

**Comment by kadirnar** [comment]
> ![Image](https://github.com/user-attachments/assets/a4c23af2-336e-4a46-bfba-267699b66812)


You should change the loss values in the config file.
https://github.com/Respaired/Tsukasa-Speech/issues/6#issuecomment-2758477322

**Comment by iAdityaVishnu** [comment]
If I decrease the learning rate than it would lead to underfitted model

**Comment by iAdityaVishnu** [comment]
Here is tensorboard link 

http://151.115.73.7/

**Comment by iAdityaVishnu** [comment]
@yl4579

**Comment by martinambrus** [comment]
A bit late to the discussion here but StyleTTS2 is currently incompatible with H100 hardware. Also, there are a few issues here mentioning NaN during training. Have a look around.
